docs/user_guide/command_line_interface.md

```diff
@@ -738,17 +742,56 @@ optional arguments:
 As the `--visualization` parameters suggests, there is a vast number of visualizations readily available.
 Each of them requires a different subset of this command's arguments, so they will be described one by one in the [Visualizations](#visualizations) section.
 
+# init_config
+
+```
+usage: ludwig init_config [options]
+
+This script initializes a valid config from a dataset.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -d DATASET, --dataset DATASET
+                        input data file path
+  -t TARGET, --target TARGET
+                        target(s) to predict as output features of the model
+  --time_limit_s TIME_LIMIT_S
+                        time limit to train the model in seconds when using hyperopt
+  --tune_for_memory TUNE_FOR_MEMORY
```

Comment:
--tune_for_memory is a boolean flag afaik. (true/false)

Reply:
You're right.

This is automatically generated from argparse, so if we want to update the strings for the boolean flags here, I should also update it in `cli.py`.

Perhaps:

```
--tune_for_memory TUNE_FOR_MEMORY
                        (bool) refine hyperopt search space based on available host / GPU memory
```

or:

```
--tune_for_memory TUNE_FOR_MEMORY
                        (true/false) refine hyperopt search space based on available host / GPU memory
```

?